src/lib_json/json_value.cpp

```diff
@@ -190,6 +191,7 @@ static inline void releaseStringValue(char* value, unsigned) {
 
 namespace Json {
 
+#if JSON_USE_EXCEPTION
```

Comment:
What if we simply guard the `throw`? We could also `abort()` on the next line. **JsonCpp** will not call these `throw*` functions when `JSON_USE_EXCEPTIONS` is false anyway.